packages/firestore/package.json

```diff
@@ -27,6 +27,7 @@
     "@types/mocha": "^2.2.42",
     "@types/sinon": "^2.3.3",
     "chai": "^4.1.1",
+    "grpc": "^1.6.6",
```

Comment:
Just to be clear: this will accept grpc 1.7 when it comes out, yes?

Reply:
Yup! I always get the tilde vs. caret thing confused:

https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json

> The caret, on the other hand, is more relaxed. It will update you to the most recent major version (the first number). ^1.2.3 will match any 1.x.x release including 1.3.0, but will hold off on 2.0.0.